crates/wdl-lint/RULES.md

```diff
@@ -7,6 +7,7 @@ This table documets all implemented `sprocket` lin rules impleemnted on the `mai
 | Name | Tags | Description |
 |:-|:-|:-|
 | `CallInputSpacing` | Spacing, Style | Detects unsorted input declarations |
```

Comment:
```suggestion
| `CallInputSpacing` | Spacing, Style | Ensures call inputs are spaced appropriately |
```
this is a bad copy+paste that existed before your PR, but please include this patch in your PR. This also needs to be updated on line:52 of `rules/call_input_spacing.rs` (should have the same text as this suggestion)